common/src/main/java/tachyon/Constants.java

```diff
@@ -364,5 +364,10 @@
   public static final short DEFAULT_TFS_FULL_PERMISSION = (short) 0777;
   public static final short FILE_DIR_PERMISSION_DIFF = (short) 0111;
 
+  public static final String SECURITY_AUTHORIZATION_PERMISSION_ENABLED =
+      "tachyon.security.authorization.permission.enabled";
```

Comment:
Oh, yes. These security related properties are all missed there... Both the finished authentication properties, and these authorization properties in development.

After go through the `Configuration-Settings.md`, I am not sure which category to put them in (some are common, but some are master specific), or create a new category say `Security`. Also the property names might need changes for consistence.

So shall we open a new JIRA to track this?